ember.md

```diff
@@ -1,7 +1,7 @@
 # Ember
 
 1. **[General Structure](#general-structure)** 
-	[ [Properties](#general-structure--properties) ]
+	[ [Properties](#general-structure--properties) [Multi-property](#general-structure--multiProperty) ]
```

Comment:
These need to be separated out into their own square brackets

Reply:
Done!